app/controllers/trainees/diversity/disability_disclosures_controller.rb

```diff
@@ -39,6 +39,7 @@ def disability_disclosure_params
 
       def redirect_to_relevant_step
         if @disability_disclosure_form.disability_not_provided? || @disability_disclosure_form.no_disability?
+          trainee.clear_disabilities
```

Comment:
Is it possible to move this inside the form object and do it in a transaction with updating the trainee? It would be ideal to only have one point of success or failure. If this line were to throw an exception then the record would be partially saved but the user would get an error.

Reply:
Done